pkg/clients/s3/bucket.go

```diff
@@ -197,6 +201,14 @@ func WebsiteConfigurationNotFound(err error) bool {
 	return false
 }
 
+// AllowedParameters is parses the aws Error and validates if the method and parameters are now allowed for a request
+func AllowedParameters(err error) bool {
```

Comment:
I'd prefer to have two error functions instead of one and have the `OR` statement in where they are necessary, i.e. `accelerateConfig.go`, but I guess that's also fine if you think these two errors happens interchangeably in all cases without difference.

Reply:
That's fair, these seem like general errors that could happen anywhere.